lib/hawkular/metrics/metrics_client.rb

```diff
@@ -43,6 +46,9 @@ def initialize(entrypoint,
                    options = {})
       entrypoint = normalize_entrypoint_url entrypoint, 'hawkular/metrics'
       super(entrypoint, credentials, options)
+      version = fetch_version_and_status['Implementation-Version']
+      major, minor = version.scan(/\d+/).map(&:to_i)
```

Comment:
any potential need in error handling? in case `fetch_version_and_status` fails and the version var is corrupted/empty?